genesis/engine/entities/rigid_entity/rigid_entity.py

```diff
@@ -352,7 +352,11 @@ def _load_MJCF(self, morph, surface):
         for i_e in range(mj.neq):
             e_info = mju.parse_equality(mj, i_e, morph.scale, ordered_links_idx)
             # only those two types of equality are supported
-            if e_info["type"] == gs.EQUALITY_TYPE.CONNECT or e_info["type"] == gs.EQUALITY_TYPE.JOINT:
+            if (
```

Comment:
I don't think it is worth checking this if there is no other choice x)

Anyway, prefer `e_info["type"] in (gs.EQUALITY_TYPE.CONNECT, gs.EQUALITY_TYPE.JOINT, gs.EQUALITY_TYPE.WELD)`